source/shadow.c

```diff
@@ -938,6 +938,12 @@ ShadowStatus_t Shadow_MatchTopic( const char * pTopic,
                                                            NULL,
                                                            NULL );
 
+    /* Update the output parameter for Thing name length.
+     * Shadow_MatchTopicString takes a pointer to a 8 bit unsigned integer for
+     * output parameter `Thing name length`, whereas Shadow_MatchTopic takes a
```

Comment:
(Minor) Does the use of ticks "`" help in any way?

Reply:
No. It can be changed. Thanks.